### DISCUSSION TITLE
SCTRL to disable Pausetime

### DISCUSSION THREAD

**Discussion by Ksylatarsh** [ikemen-engine/Ikemen-GO#1235] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Pausetime on Hitdef can sometimes be annoying, specially when trying to switch animations over and trying to apply velocity/acceleration since ignorehitpause doesn't work on these specifically, so a SCTRL to fully disable it on call like "hitpausedisable" would be greatly appreciated instead of having to circumvent it with workarounds, could even be an assertspecial (this one would fit in better).

### Describe the solution you'd like

Make a new SCTRL or AssertSpecial to get rid of it when called, instead of ignoring it.

### Additional context

Custom Cornerpush systems would be greatly benefited from this.

### Search terms

hitdef pausetime hitpause sctrl

**Comment by potsmugen** [comment]
A sctrl to modify* hitpause might be useful (and easy to implement), but it begs the question: why do you have a hitpause if you want to move during hitpause?

*I'd personally do it like "hitpauseset" or something. You could set it to anything that way (including 0).

**Reply by Ksylatarsh** [reply]
Yeah I agree modify it would be way better than removing it entirely.

**Comment by potsmugen** [comment, marked as answer]
Already implemented. `ModifyPlayer` can change `hitpausetime`.
